### PR TITLE
feat(#1187) Comments for `.gmi` files

### PR DESCRIPTION
**Pull request by volodya-lombrozo**
I've added different type of comments for each `.gmi` file type. 
Closes: #1187

### CODE REVIEW

**Comment by volodya-lombrozo** [comment]
@yegor256 take a look, please.

**Comment by volodya-lombrozo** [comment]
@mximp take a look, please.

File: eo-maven-plugin/src/main/resources/org/eolang/maven/gmi-to/license.xsl [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="license" version="2.0">
```
Comment (mximp) [inline eo-maven-plugin/src/main/resources/org/eolang/maven/gmi-to/license.xsl:25]: @volodya-lombrozo should this be in `gmi-to` package? can we move it to `gmi`?

Comment (volodya-lombrozo) [reply eo-maven-plugin/src/main/resources/org/eolang/maven/gmi-to/license.xsl]: @mximp you right. I've moved the `license.xsl` upper to `org/eolang/maven` and reused it in `to-java.xsl`

**Comment by volodya-lombrozo** [comment]
@mximp applied your suggestions. Please, check.

**Review by mximp — APPROVED** [review]

**Comment by volodya-lombrozo** [comment]
@yegor256 merge, please

**Pull request merged**